src/types.ts

```diff
@@ -58,15 +58,19 @@ export interface Column<TRow, TSummaryRow = unknown> {
 
 export interface CalculatedColumn<TRow, TSummaryRow = unknown> extends Column<TRow, TSummaryRow> {
   idx: number;
-  width: number;
```

Comment:
I am not sure if any formatter/editor uses width/left or not.  What is the alternative if those values are needed? ref?

Reply:
We could pass them in `editorProps` if it's ever needed.